examples/api/lib/widgets/magnifier/cupertino_text_magnifier.0.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+
+
+void main() => runApp(const TextMagnifierExampleApp(text: 'Hello world!'));
+
+class TextMagnifierExampleApp extends StatefulWidget {
```

Comment:
This sample doesn't have `TextMagnifierExampleApp` unlike the other sample in this PR. Please update this sample to be consistent.

Reply:
I have tried to keep the pattern based on the other samples, but not sure if I got the issue.